### ISSUE TITLE
Unable to locate message source for category 'sprout-module-transactional'.

### ISSUE THREAD

**Issue by alexanderloewe** [barrelstrength/sprout#272]
### Description

Redirects Cause this error when I try to edit an entry type of a section

```
Invalid Configuration – [yii\base\InvalidConfigException](https://www.yiiframework.com/doc-2.0/yii-base-invalidconfigexception.html)
Unable to locate message source for category 'sprout-module-transactional'.
1. in /home/public_html/vendor/yiisoft/yii2/i18n/I18N.phpat line 201
192193194195196197198199200201202203        if (isset($this->translations['*'])) {
            $source = $this->translations['*'];
            if ($source instanceof MessageSource) {
                return $source;
            }
 
            return $this->translations[$category] = $this->translations['*'] = Yii::createObject($source);
        }
 
        throw new InvalidConfigException("Unable to locate message source for category '$category'.");
    }
}
```

### How to reproduce

When trying to edit an entry type

### Sprout Version

4.46.0

### Craft Version

4.5.3

### Database Type Version

MySQL 5.7.41

**Comment by BenParizek** [comment]
I've just released Redirects v4.46.1 which should resolve this.

I've also put a test in place to help avoid issues like this in the future. Thanks for reporting and let me know if you run into anything else.

**Comment by alexanderloewe** [comment]
Thanks, now everything works fine again ;)
